docs/src/components/productX/XRoadmap.tsx

```diff
@@ -0,0 +1,190 @@
+import PendingActionsRounded from '@mui/icons-material/PendingActions';
+import SpeedRounded from '@mui/icons-material/SpeedRounded';
+
+export default function XRoadmap() {
+  function renderList(content: React.ReactElement) {
+    return (
+      <Box
+        sx={{
+          mt: 2,
+          display: 'grid',
+          alignItems: 'center',
+          gap: 2,
+          gridTemplateColumns: 'max-content 1fr',
+        }}
+      >
+        {content}
+      </Box>
+    );
+  }
+  const bullet = (
+    <Box
+      sx={{
+        ml: 1,
+        mr: -2,
+        display: 'flex',
+        alignItems: 'center',
+        '&:before': {
+          content: '""',
+          display: 'block',
+          height: 2,
+          width: 15,
+          bgcolor: 'primaryDark.400',
+        },
+        '&:after': {
+          content: '""',
+          width: 6,
+          height: 6,
+          bgcolor: 'warning.500',
+          borderRadius: '50%',
+          display: 'block',
+          marginRight: 1.5,
+        },
+      }}
+    />
+  );
+  return (
+    <ThemeProvider theme={brandingDarkTheme}>
+      <Section bg="dim">
+        <Box sx={{ py: 4 }}>
+          <Grid container spacing={2} alignItems="center">
+            <Grid item xs={12} md={6}>
+              <Box maxWidth={500}>
+                <SectionHeadline
+                  overline="Roadmap"
+                  title="Follow the MUI X roadmap for future updates"
+                  description="It's just the beginning of the MUI X components. Stay tuned for the exciting news and updates coming soon!"
+                />
+                <Button
+                  component={Link}
+                  href={ROUTES.roadmap}
+                  noLinkStyle
+                  size="large"
+                  variant="contained"
+                  endIcon={<KeyboardArrowRightRounded />}
+                >
+                  See the roadmap
+                </Button>
+              </Box>
+            </Grid>
+            <Grid item xs={12} md={6} sx={{ minWidth: 0 }}>
+              <Box
+                sx={{
+                  display: 'grid',
+                  mt: { xs: 2, sm: 0 },
+                  overflow: { xs: 'auto', lg: 'unset' },
+                  gap: 2,
+                  alignItems: 'flex-start',
+                  gridTemplateColumns: { xs: 'repeat(1)', lg: 'repeat(3, minmax(200px, 1fr))' },
+                  typography: 'body2',
+                  '& .MuiPaper-root': {
+                    p: 2,
+                    bgcolor: 'primaryDark.600',
+                  },
+                  '& svg': {
+                    color: 'primary.300',
+                  },
+                }}
+              >
+                <Paper variant="outlined">
+                  <Box sx={{ fontWeight: 'bold' }}>In the lab:</Box>
+                  <Box sx={{ color: (theme) => theme.palette.text.secondary }}>
+                    Almost ready to go.
+                  </Box>
```

Comment:
What do you think about this version instead? It emphasizes that it can be used, but will be improved. The current version might be seen as an emphasis on that it's not really ready to be used. 

```suggestion
                  <Box sx={{ fontWeight: 'bold' }}>Released</Box>
                  <Box sx={{ color: 'text.secondary' }}>
                    More updates coming.
                  </Box>
```

<img width="670" alt="Screenshot 2021-09-05 at 14 26 50" src="https://user-images.githubusercontent.com/3165635/132126767-0de40b1f-16ad-4ca3-947c-2c23eb8dd98c.png">

Reply:
I'm ok with it, agree with your point of view!